src/test/kotlin/lotto/WinningStaticsTest.kt

```diff
@@ -5,25 +5,91 @@ import io.kotest.matchers.shouldBe
 
 class WinningStaticsTest : FreeSpec({
 
     "result" - {
 
-        "당첨 로또 개수와 수익률을 반환한다." {
+        "당첨된 로또가 없는 경우 결과를 반환한다." {
+            val lottoNumberSet = LottoNumberSet(listOf(13, 14, 15, 16, 17, 18))
+            val lottoRecord = listOf(lottoNumberSet)
+            val paymentDto = InputPaymentRequestDto.convertPayment("14000")
+            val winningLotto = InputLottoNumberDto.of(listOf("1", "2", "3", "10", "11", "12"), "13")
+            val winningStat = WinningStatics(paymentDto, winningLotto, lottoRecord)
+
+            val winningResult = winningStat.result()
+
+            LottoWinningAmount.values().forEach { it ->
+                winningResult.winningLottoList[it] shouldBe 0
+            }
+            winningResult.profitRatio shouldBe 0
+        }
+
+        "당첨 로또가 3개인 경우 5등입니다." {
             val lottoNumberSet = LottoNumberSet(listOf(1, 2, 3, 4, 5, 6))
             val lottoRecord = listOf(lottoNumberSet)
-            val paymentDto = InputPaymentRequestDto.convertToOperand("14000")
-            val winningLotto = InputWinningLottoNumberDto.convertOperand(listOf("1", "2", "3", "10", "11", "12"))
+            val paymentDto = InputPaymentRequestDto.convertPayment("14000")
+            val winningLotto = InputLottoNumberDto.of(listOf("1", "2", "3", "10", "11", "12"), "13")
+            val winningStat = WinningStatics(paymentDto, winningLotto, lottoRecord)
+
+            val winningResult = winningStat.result()
+
+            winningResult.winningLottoList[LottoWinningAmount.FIFTH] shouldBe 1
+            winningResult.profitRatio shouldBe 0.35
+        }
+
+        "당첨 로또가 4개인 경우 4등입니다." {
+            val lottoNumberSet = LottoNumberSet(listOf(1, 2, 3, 10, 5, 6))
+            val lottoRecord = listOf(lottoNumberSet)
+            val paymentDto = InputPaymentRequestDto.convertPayment("14000")
+            val winningLotto = InputLottoNumberDto.of(listOf("1", "2", "3", "10", "11", "12"), "13")
             val winningStat = WinningStatics(paymentDto, winningLotto, lottoRecord)
 
             val winningResult = winningStat.result()
 
             winningResult.winningLottoList[LottoWinningAmount.FOURTH] shouldBe 1
+            winningResult.profitRatio shouldBe 3.57
+        }
 
-            winningResult.profitRatio shouldBe 0.35
+        "당첨 로또가 보너스는 일치하지 않는 5개인 경우 3등입니다." {
+            val lottoNumberSet = LottoNumberSet(listOf(1, 2, 3, 10, 11, 6))
+            val lottoRecord = listOf(lottoNumberSet)
+            val paymentDto = InputPaymentRequestDto.convertPayment("14000")
+            val winningLotto = InputLottoNumberDto.of(listOf("1", "2", "3", "10", "11", "12"), "20")
+            val winningStat = WinningStatics(paymentDto, winningLotto, lottoRecord)
+
+            val winningResult = winningStat.result()
+
+            winningResult.winningLottoList[LottoWinningAmount.THIRD] shouldBe 1
+            winningResult.profitRatio shouldBe 107.14
+        }
+
+        "당첨 로또가 보너스가 일치하는 5개인 경우 2등입니다." {
```

Comment:
```suggestion
        "당첨 로또가 5개이면서 보너스 볼이 일치하는 경우 2등입니다." {
```
이렇게 작성하면 조금 더 잘 이해가 될 것 같아요.